pallets/collator-selection/src/lib.rs

```diff
@@ -239,6 +248,7 @@ pub mod pallet {
 	#[pallet::error]
 	pub enum Error<T> {
 		TooManyCandidates,
+		TooFewCandidates,
 		Unknown,
 		Permission,
 		AlreadyCandidate,
```

Comment:
its nice to have doc comments on these as they appear in the metadata and show up in the UI